js/matchkeys/goldrush/goldrush.mjs

```diff
@@ -1,7 +1,7 @@
 // Generates GoldRush match key.
 
-function loadMarcJson(marcJson) {
-  const marcObj = JSON.parse(marcJson);
+function loadMarcJson(record) {
+  const marcObj = JSON.parse(record).marc;
```

Comment:
@dcrossleyau @adamdickmeiss I think to avoid this we should assume the that matchkey functions accepts an already parsed JSON and call JSON.parse(x).marc in the "script" section.

Reply:
Agree, that seems cleaner

Reply:
Only that we can't combine the "script" with "url" in the ES-module mode -- I tried to no avail. I think we should merge this as-is and I will reach out to GraalVM folks to ask if it is possible to combine the two somehow.